src/Workspaces/Remote/ServiceHub/Host/SolutionCreator.cs

```diff
@@ -1,534 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
in general, i recommend not renamign the file in this main PR.  this makes the diff easier.  then just have a quick followup pr that renames.

Reply:
Tried to make it easier in a separate commit.